custom_components/ha-glocaltokens/const.py

```diff
@@ -27,13 +31,42 @@
 CONF_MASTER_TOKEN = "master_token"
 
 # Defaults
-DEFAULT_NAME = "GlocalToken"
+DEFAULT_NAME = "Glocaltokens"
+
+# API
+PORT = 8443
+ENDPOINT = '/setup/assistant/alarms'
```

Comment:
Let's name it `API_ENDPOINT_ALARMS`, because as time goes we might introduce more and more API URLs and need to have a consistent way of naming them